composer.json

```diff
@@ -51,5 +54,10 @@
         "branch-alias": {
             "dev-master": "2.1-dev"
         }
-    }
+    },
+    "conflicts":{
```

Comment:
correct spelling would be `conflict` without the s at the end: https://getcomposer.org/doc/04-schema.md#conflict

unfortunately composer does not complain when it encounters unknown keys in the file.